Resources/Prototypes/_CM14/Datasets/tips.yml

```
@@ -0,0 +1,218 @@
+  - "As Xeno, you can break Night-Vision goggles that some marines wear on their helmets. Just aim for the head and slash until the goggles shatter."
+  - "Pounces are ineffective on marines who are laying down."
+  - "You may rest immediately during a pounce to pounce straight through mobs. It's not very practical or useful though."
+  - "Pouncing someone who is buckled to a chair will still stun them, but you won't jump into their tile and they will not be knocked to the ground."
+  - "Star shell dust from said grenades is just as meltable as normal flares."
+  - "Playable Facehuggers can leap onto targets with a one-second windup, but this will only infect them if they are adjacent to it.  Otherwise, it will simply knock them down for a small duration."
+  - "As a Facehugger, you cannot talk in hivemind, but you can still open Hive Status and overwatch your sisters. This can be useful if you're locating other Facehuggers, flanker castes, or trying to learn from experienced Facehugger players."
+  - "Shift-clicking the Queen indicator will tell you what area you are in, on the map."
+  - "As a Ravager your abilities become greatly enhanced when you empower with three or more people around you."
+# Meta Tips
+  - "Try not to get too mad about dying. We’re all here to have fun."
+  - "After dying, ask yourself what you did wrong and make a mental note to not make the same mistake again."
+  - "Communication, be it from a marine to a marine, a drone to the queen, or command to everyone, is vital and information on flanks can change how the entire round plays out."
+  - "As an alien or marine, be careful of the flank, regardless of if the push is going well or stalling out."
+  - "Half of getting good is knowing to be aggressive. The other half is knowing when not to be aggressive."
+  - "Use 'North, South, West, East' when referring to locations in-game rather than 'up, down, left, right'."
+  - "As a mentor, you can become the imaginary friend of a new player to teach them!"
+  - "You shouldn't ignore what your allies are up to. Sometimes they can be organizing a flank in hivemind/radio, sometimes they can be walking up behind you with a slug-loaded shotgun. Either way, it pays to be alert to what they're doing, as much to as what the enemies are."
+  - "You can always AdminHelp with the F1 key to question a member of staff regarding rules or game bugs."
+  - "As ghost you are given extra tools for spectating the round. You can jump and follow specific players, get notifications about CAS and OB strikes, can see all health bars, and such."
+  - "You can press ESC key to bring up the game pause menu. It allows you change settings, AdminHelp and MentorHelp, and even access the Web Maps of game by clicking at top right."
+  - "Control of intelligence is important be it for retrieving it as marine, or denying it as Xeno."
+  - "If the lobby music is too loud or bothering you, you can disable it in Preferences tab at top right."
```

This one's SS13 specific, adapted to SS14 it would look like this:
```suggestion
  - "If the lobby music is too loud or bothering you, you can disable it in Options > Audio in the Escape menu."
```